stock_prices/stock_prices.py

```diff
@@ -2,14 +2,25 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    current_min_price_so_far = prices[0]
+    max_profit_so_far = -100000
+    for i in range(1, len(prices) - 1):
```

Comment:
good job with this one. You can find it without a double for loop by keeping track of the max profit while the first for loop iterates through the array.

min_price = prices[0]
max_profit = prices[1] - min_price

for i in range(1, len(prices)):
____price = prices[i]
____max_profit = max(price - min_price, max_profit)
____min_price = min(price, min_price)

return max_profit